src/Microsoft.DotNet.Build.Tasks/PackageFiles/VersionTools.targets

```diff
@@ -29,6 +35,7 @@
     <UpdateDependencies DependencyBuildInfo="@(DependencyBuildInfo)"
                         ProjectJsonFiles="@(ProjectJsonFiles)"
                         XmlUpdateStep="@(XmlUpdateStep)"
+                        BuildInfoCacheDir="$(BuildInfoCacheDir)"
```

Comment:
What does "UpdateDependencies" do with the cached information?  Shouldn't it always read from @(DependencyBuildInfo)?